src/components/sidebar/components/sidebar-nav/index.tsx

```diff
@@ -18,14 +23,29 @@ const SidebarNav: React.FC<SidebarNavProps> = ({ menuItems, title }) => (
       Skip to main content
     </a>
     <ul className={s.sidebarNavList}>
-      {menuItems.map((item, index) => (
-        <SidebarNavMenuItem
-          item={item}
-          // TODO: use item.id when it has been added to the metadata (see above)
-          // eslint-disable-next-line react/no-array-index-key
-          key={`sidebar-nav-menu-item-${index}`}
-        />
-      ))}
+      {menuItems.map((item, index) => {
+        // TODO: this is stopgap solution, I think there may
+        // TODO: be some benefit to passing basePath explicitly rather
+        // TODO: than trying to parse it from the current route,
+        // TODO: as it allows us to render the nav outside of its typical
+        // TODO: location, which seems like something we'll likely do
+        // TODO: for the initial launch of the /{product} home page.
+        // TODO: One option, as below, is to make basePath optional,
+        // TODO: and if it is present, then generate the href from the
+        // TODO: basePath rather than try to determine the necessary
+        // TODO: item.path prefix from the window location (as currently
+        // TODO: done in getItemPath() if item.href is not present).
+        const href =
+          item.path && basePath ? basePath + '/' + item.path : item.href
+        return (
+          <SidebarNavMenuItem
+            item={{ ...item, href }}
```

Comment:
As a heads up, `SidebarNavMenuItem` will be using the `path` and `href` properties to distinguish between internal and external links, respectively. (Working on this now). So if we introduce a `basePath` here, then I think it needs to be passed through to `SidebarNavItem` and `getItemPath` needs to be updated to use `basePath` if it exists instead of the current route.

Reply:
👍 Thanks for flagging! Will see if I can get that working (just noticed the changes I've made so far don't seem to be working, ahah 😭 )

Reply:
@ashleemboyer with the latest Figma mocks, which have custom content in the `/waypoint` view sidebar, was able to sidestep having to add `basePath` for now, and have gone with `href` for demo purposes. Have shimmed in an external link icon functionality separate to `href`, but definitely open to completely undoing that post-demo.

Have mulled this over a bit, and am thinking a longer-term solution here might be to update our server-side docs page or layout functions so that the returned `navData` already has the correct routes built in to `path` (ie `navData` contains paths like `/waypoint/docs/intro` rather than `/intro`). This way we could avoid having to complete the paths on the client-side using router utilities etc, and the `sidebar-nav` component might become a more flexible display component rather than having route-specific logic.... not a fully formed idea yet, but tossing it out there anyways ☺️